openapi/platform/platform.yaml

```diff
@@ -4338,36 +4914,7 @@ components:
             lineItems:
               type: array
               items:
-                type: object
-                properties:
-                  sku:
-                    type: string
-                    description: 'The unique identifier of a product, product edition or package. It can be obtained by following [this guide](https://docs.apigateway.co/docs/openapi-specs/docs/Guides/Sell/FindSKU.md).'
```

Comment:
I missed it before merging but this refactor resulted in changing the field name from `sku` to `id` for the existing salesOrders resource.

Reply:
Do we want to revert that?

Reply:
Just make another PR fixing it. It is only like 2 lines out of this whole PR so not worth reverting it.

Reply:
Oh okay I also got the fix in this [draft PR ](https://github.com/vendasta/api-gateway-docs/pull/105)as well. Im doing a pass on the proposal builder docs